hack/start_flotta_k8s.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+export OPERATOR_PATH="<path to flotta operator repo>"
+export DEVICE_WORKER_PATH="<path to flotta device worker repo>"
+export YGGDRASIL_PATH="<path to yggdrasil repo>"
+
+# Start flotta operator
+gnome-terminal --tab -- bash -c \
+        "cd $OPERATOR_PATH; \
+        IMG=<image registry and tag> make docker-build docker-push; \
+        IMG=<image registry and tag> TARGET=k8s make deploy; \
+        exec bash"
+
+# Start flotta device worker
+gnome-terminal --tab -- bash -c \
+        "cd $DEVICE_WORKER_PATH; \
+        make uninstall; \
+        make install; \
+        exec bash"
+
+# Start yggdrasil
+gnome-terminal --tab -- bash -c \
+        "cd $YGGDRASIL_PATH; \
+        git checkout cdde836c519ac72302714b5f7ae90e45de7c79a7; \
+        export PREFIX=/var/local; \
+        make bin; \
+
+        sudo ./yggd \
+          --log-level info \
+          --protocol http \
+          --path-prefix api/flotta-management/v1 \
+          --client-id $(cat /etc/machine-id) \
+          --cert-file /etc/pki/consumer/cert.pem \
```

Comment:
these certs are not correct, no? should be the ones from make get-certs

Reply:
+1. and don't forget the `sudo chown root.root /tmp/*.pem` pls

Reply:
Do you mean
```
--cert-file /tmp/cert.pem
--key-file /tmp/key.pem
```
?

Reply:
Is the `--ca-root /tmp/ca.pem` necessary?